packages/web/src/components/cells/code.tsx

```diff
@@ -434,6 +453,22 @@ function Header(props: {
                 <TooltipContent>Edit cell using AI</TooltipContent>
               </Tooltip>
             </TooltipProvider>
+            <TooltipProvider>
+              <Tooltip>
+                <TooltipTrigger asChild>
+                  <Button
+                    variant="icon"
+                    size="icon"
+                    disabled={cellMode !== 'off'}
+                    onClick={onFormatCode}
+                    tabIndex={1}
+                  >
+                    <CodeXml size={16} />
```

Comment:
Couple notes on this:

* I wonder if there's a common icon for code formatting? This icon is ambiguous and could be used to indicate many different things in a product like ours. Maybe [braces](https://lucide.dev/icons/braces), but that doesn't feel much different?
* This icon feels a bit odd with the current debounce UX. You have < 2 seconds after typing anything before this pressing this icon will have no affect. That said, it is useful if you import a srcbook and want to format the cell without typing in it first.

Reply:
I like, and I think I've seen this before, using a brush icon:
![CleanShot 2024-09-01 at 13 33 43](https://github.com/user-attachments/assets/4bed6eb2-d8b9-414f-8dc0-eb7e1e48bde0)

`PaintbrushVertical` in lucide-react.